book/src/clauses/well_known_traits.md

```diff
@@ -0,0 +1,57 @@
+# Well known traits
+
+Not all traits can be encoded in Rust's type system: special traits
```

Comment:
I think a better phrasing would be: "Not all trait logic can be encoded in Rust types: ..."